### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by oricho123**

### CODE REVIEW

File: quickstart-python/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Monday-Code Python Quickstart
+
+This is a Python project that uses Flask as a web framework. The project is structured around a service-oriented
+architecture, with different services handling different aspects of the application.
```
Comment (oricho123) [inline quickstart-python/README.md:4]:
```suggestion
This Python Flask application is designed to exemplify the practical use of the Monday Code platform by integrating various Monday app functionalities. It's an essential resource for developers looking to leverage Monday Code features through a Python environment.
```

File: quickstart-python/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Monday-Code Python Quickstart
+
+This Python Flask application is designed to exemplify the practical use of the Monday Code platform by integrating various Monday app functionalities.
+It's an essential resource for developers looking to leverage Monday Code features through a Python environment.
```
Comment (oricho123) [inline quickstart-python/README.md:4]: I feel like some refinement is needed here

File: quickstart-python/.env.example [hunk 3]
```diff
@@ -0,0 +1,5 @@
+MONDAY_OAUTH_CLIENT_ID=d2example88893415a4783b23e3a4fa4b
+MONDAY_OAUTH_CLIENT_SECRET=d3bd4547example6350b79f09b664c18
+MONDAY_OAUTH_TOKEN_PATH=https://auth.monday.com/oauth2/token
+MONDAY_SIGNING_SECRET=b8bbb24beexamplebbc49d2247e
+MONDAY_OAUTH_BASE_PATH=https://auth.monday.com/oauth2/authorize
```
Comment (gregra81) [inline quickstart-python/.env.example:5]: 1. Don't commit
2. End each file with a line break

```suggestion
MONDAY_OAUTH_CLIENT_ID=
MONDAY_OAUTH_CLIENT_SECRET=
MONDAY_OAUTH_TOKEN_PATH=
MONDAY_SIGNING_SECRET=
MONDAY_OAUTH_BASE_PATH=

```

Comment (oricho123) [reply quickstart-python/.env.example:5]: All of them are examples, not real values

File: quickstart-python/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+The project includes the following services:
+
+- `MondayApi`: Handles interactions with the Monday API.
+- `JWTService`: Handles JSON Web Token operations.
+- `SecretService`: Manages secrets within the application.
+- `StorageService`: Manages storage operations.
+- `QueueService`: Manages queue operations.
+- `SecureStorage`: Handles secure storage operations.
+- `MailService`: Handles mail operations.
```
Comment (gregra81) [inline quickstart-python/README.md:98]: This creates a false believe that we support sending emails OOTB as part of monday code utils

File: quickstart-python/.env.example (hunk 3 above)
Comment (gregra81) [reply quickstart-python/.env.example:5]: I'd keep `MONDAY_OAUTH_TOKEN_PATH` and `MONDAY_OAUTH_BASE_PATH` and remove the values of the rest, i.e.
```suggestion
MONDAY_OAUTH_TOKEN_PATH=https://auth.monday.com/oauth2/token
MONDAY_OAUTH_BASE_PATH=https://auth.monday.com/oauth2/authorize
MONDAY_OAUTH_CLIENT_ID=
MONDAY_OAUTH_CLIENT_SECRET=
MONDAY_SIGNING_SECRET=

```

File: quickstart-python/SETUP.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        1. Install apps-cli:
+           ```bash
+           $ npm i -g @mondaycom/apps-cli
+           ```
+        2. Set secrets:
+           ```bash
+           $ mapps code:env
+           ```
+           Choose the app to set environment variables > set > enter the secret key > then enter the value of the
+           secret. The secret will be injected into process.env of your deployment.
```
Comment (gregra81) [inline quickstart-python/SETUP.md:53]: IMO we should direct the developers to use the UI in the dec center for setting secrets

File: quickstart-python/SETUP.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+### Deploying the App to Monday Code
+
+1. Go to the `General` tab under monday code, click on `Connect monday code` and follow the instructions on the screen.
+2. Choose you integration Feature Under the `Feature` tab, create a new build and choose monday code as the deployment
+   target.
+3. Reconfigure the OAuth redirect URL with the new URL provided by Monday Code.
+
+That's it! You have successfully deployed the app to Monday Code. 🚀
+
+### Setting up the Environment Variables
```
Comment (gregra81) [inline quickstart-python/SETUP.md:40]: This is great, but better to put it in our docs, @yuhgto

File: quickstart-python/SETUP.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+           ```bash
+           $ npm i -g @mondaycom/apps-cli
+           ```
+        2. Set secrets:
+           ```bash
+           $ mapps code:env
+           ```
+           Choose the app to set environment variables > set > enter the secret key > then enter the value of the
+           secret. The secret will be injected into process.env of your deployment.
+    2. instead, you can set the secrets manually in monday-code by going to the `General` tab under monday code and
```
Comment (gregra81) [inline quickstart-python/SETUP.md:54]: 2. should be the first one is it is the preferred option

**Review by gregra81 — APPROVED** [review]
Amazing 👏

**Pull request merged**